Exercise-4/flattenArray.js

```diff
@@ -0,0 +1,39 @@
+// Function that checks the level of nesting of an array and flattens the array into a single-dimension array
+
+const nestedArray1 = [1, 2, [3, 4, [5, 5]]];
+
+const nestedArray2 = [1, 2, [3, 4, [5, 6, [7, 8, [9, 10, [11, 12]]]]]];
+
+const maxDepth = (inputArray, depth = 1) => {
```

Comment:
The function name should start with a verb, for example, `getMaxDepth`. maxDepth alone feels like a standard variable